synapse/models/belief.py

```diff
@@ -0,0 +1,75 @@
+import asyncio
+import logging
+
+import synapse.exc as s_exc
+
+import synapse.lib.types as s_types
+import synapse.lib.module as s_module
+import synapse.lib.version as s_version
+
+logger = logging.getLogger(__name__)
+
+class BeliefModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('belief', {
+            'types': (
+
+                ('belief:system', ('guid', {}), {
+                    'doc': 'A belief system such as an ideology, philosophy, or religion.'}),
+
+                ('belief:system:type:taxonomy', ('taxonomy', {}), {
+                    'interfaces': ('taxonomy',),
+                    'doc': 'A hierarchical taxonomy of belief systems.'}),
+
+                ('belief:tenet', ('guid', {}), {
+                    'doc': 'A concrete tenet potentially shared by multiple belief systems.'}),
+
+                ('belief:subscriber', ('guid', {}), {
+                    'doc': 'A contact which subscribes to a belief system.'}),
+            ),
+            'forms': (
+
+                ('belief:system', {}, (
+
+                    ('name', ('str', {'onespace': True, 'lower': True}), {
+                        'doc': 'The name of the belief system.'}),
+
+                    ('type', ('belief:system:type:taxonomy', {}), {
+                        'doc': 'A taxonometric type for the belief system.'}),
+
+                    ('began', ('time', {}), {
+                        'doc': 'The time that the belief system was first observed.'}),
+
+                )),
+
+                ('belief:tenet', {}, (
```

Comment:
Can we add :desc?

Reply:
I think tenets should be straightforward enough not to warrant a :desc prop?

Reply:
We can discuss, but i'm not gonna hold this PR for it :)